frontend/src/pages/Broadcast/Component/Carousel/carousel.module.scss

```diff
@@ -29,13 +29,15 @@
 
 .slide-card {
   position: relative;
-  min-height: 31em;
-  min-width: 20em;
+  height: 31em; // Fixed height for the cards
```

Comment:
fixing height and width is making it look bad, width and height can be kept responsive, and instead of making the content scrollable, should only first 200 characters and end with `...` , users can anyway see its detail if they wish

<img width="1457" alt="Screenshot 2024-07-03 at 11 01 30 PM" src="https://github.com/HITK-TECH-Community/Community-Website/assets/44888949/e4901f04-b87a-4a8e-9ee5-55fb839fe6c8">